dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/RepositoryUtility.java

```diff
@@ -122,9 +122,8 @@ static DefaultRepositorySystemSession createDefaultRepositorySystemSession(
    * customary ~/.m2 directory. If not found, it creates an initially empty repository in
    * a temporary location.
    */
-  public static RepositorySystemSession newSession(RepositorySystem system) {
+  public static DefaultRepositorySystemSession newSession(RepositorySystem system) {
     DefaultRepositorySystemSession session = createDefaultRepositorySystemSession(system);
-    session.setReadOnly();
```

Comment:
I think setting we want to set the maven directory in createDefaultRepositorySystemSession to keep the returned session read-only.

Reply:
I'm not sure that's necessary. Is there some reason it has to be read-only?

I did try setting it here, but the extra code to configure it from the test was even uglier since DependencyGraphBuilderIntegrationTest doesn't directly access RepositoryUtility.

Reply:
OK. Then this current code is good.